react/utils/webAppIndexedDB.js

```diff
@@ -0,0 +1,9 @@
+import { openDB } from 'idb' 
```

Comment:
Since this is such a small data to storage, I would use localStorage instead.

Also, it would avoid adding this lib KBs in our bundle just for this small check.

Reply:
The problem is because service worker also needs to access this data and service worker can't access local storage